src/electron/views/renderer-initializer.ts

```diff
@@ -164,16 +169,19 @@ getPersistedData(indexedDBInstance, indexedDBDataKeysToFetch).then((persistedDat
 
     const fixInstructionProcessor = new FixInstructionProcessor();
 
+    const cardSelectionMessageCreator = new CardSelectionMessageCreator(dispatcher);
```

Comment:
would it be better to put the action creator close the the action message creators as we have the user config action/message creators now? They are somehow related... maybe have them close togheter makes it easier to read/understand